packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -194,10 +242,21 @@ const metadata: Meta<TableArgs> = {
             table: {
                 disable: true
             }
+        },
+        getColumnSortData: {
+            table: {
+                disable: true
+            }
         }
     },
     args: {
         data: ExampleDataType.simpleData,
+        sortedColumns: [
+            {
+                columnId: 'first-name-column',
+                sortDirection: TableColumnSortDirection.ascending
+            }
+        ],
```

Comment:
We should avoid having such complex args in a story. Should make it really easy for users to toggle with the parameters. I think we should avoid the array and obejct types altogether. It also invents new configuration apis that don't exist outside of storybook which can be misleading.

I'd suggest having a sort-direction configuration that just manipulates, for example, the first sortable column. And then maybe a sortIndex configuration that lets you select the first sortable column.

Also, since sorting isn't even a table api, maybe we should have a separate story for the column apis? Maybe under /table-column/tests/table-column.stories.ts?

Reply:
So, this change was due to a concern I raised. My feeling was that it would be useful for users to see the behavior in the table when multiple columns had their `sortDirection/sortIndex` set in various ways. To combat the confusion you mention we created a description that was meant to explain what the editor was doing, and that it wasn't a `Table` API, and that they should view the markup to see what the editor was changing.

Reply:
That's fine, just don't make the user do that configuration, make it easy to jump between them. Another proposal could be to make a radio of interesting options to choose from:
- sort col1 asc
- sort col1 desc
- sort col1 asc and col2 desc

Show me the interesting options I should be aware of. And I can compare them using the show code feature. Don't make me write different JSON blobs to try it.

Reply:
The idea of table-column.stories.ts is a good one and will probably come with #1044. When we get to that we may want to consider moving this documentation, but I'm personally ok with it being in the table story for now, especially if we make it more usable via the suggestions above.

Reply:
I don't think it's that much overhead to split out the stories in this PR but don't feel strongly either way. Could also be a follow-up so picket fencing / naming of stuff doesn't hold up this one. Or do it all later. All good.

Reply:
I've updated the story to have three different sort options that are represented as radio buttons.